### ISSUE TITLE
Error in handling node version 7.1x

### ISSUE THREAD

**Issue by tinovyatkin** [babel/babel-preset-env#322]
```
babel-preset-env: `DEBUG` option

Using targets:
{
  "node": 7.7
}

Modules transform: commonjs

Using plugins:
  syntax-trailing-function-commas {"node":7.7}

Using polyfills:
  es7.string.pad-start {"node":7.7}
  es7.string.pad-end {"node":7.7}
```

and newer 7.10 is threatening like 7.1, and even non-existent 7.11 is below:
```
babel-preset-env: `DEBUG` option

Using targets:
{
  "node": 7.11
}

Modules transform: commonjs

Using plugins:
  transform-async-to-generator {"node":7.11}
  syntax-trailing-function-commas {"node":7.11}

Using polyfills:
  es7.string.pad-start {"node":7.11}
  es7.string.pad-end {"node":7.11}
```

**Comment by existentialism** [comment]
@tinovyatkin thanks for the issue! we've actually [already implemented this](https://github.com/babel/babel-preset-env/pull/231) in the soon to be released 2.x version. (Also ref: https://github.com/babel/babel-preset-env/issues/187, https://github.com/babel/babel-preset-env/issues/261, https://github.com/babel/babel-preset-env/issues/315)

**Comment by logicoder** [comment]
FYI: obviously also ```"node": "current"``` on Node.js 7.10 leads to the same, in my case AsyncToGenerator use for Async/Await.

As a temporary workaround put a safe 7.9 or eventually 8 as the version to compile to.

BTW: loving `babel-preset-env` since first sight! thanks!

**Comment by hzoo** [comment]
We're going to try releasing a new version later today!

**Comment by existentialism** [comment]
@tinovyatkin @logicoder just released [v1.5](https://github.com/babel/babel-preset-env/releases/tag/v1.5.0) that should resolve this!

**Comment by logicoder** [comment]
Solved! 🎉 

```json
  "presets": [
    ["env", {
      "targets": {
        "node": "current"
      },
      "debug": true,
      "useBuiltIns": true
    }]
  ],
```

```
babel-preset-env: `DEBUG` option

Using targets:
{
  "node": "7.10"
}

Modules transform: commonjs

Using plugins:
  syntax-trailing-function-commas {"node":"7.10"}

Using polyfills:
  es7.string.pad-start {"node":"7.10"}
  es7.string.pad-end {"node":"7.10"}
```
